libs/core/src/components/sage-tabs/sage-tabs.tsx

```diff
@@ -51,15 +51,16 @@ export class SageTabs {
     }
   }
 
-  @Listen('keydown', {passive: true, target: 'window'})
+  @Listen('keydown', {})
   handleKeyDown(ev: KeyboardEvent) {
     const keySet = ["ArrowLeft", "ArrowRight", "Home", "End"];
```

Comment:
This is more out of curiosity than an actual issue: on my machine, `HOME` and `END` also affect the window scroll in addition to the tabs. Might be due to Storybook's app frame/iframe, though

Reply:
Thats a good callout! I'll add a `preventDefault()` to our logic.

Reply:
@monkeypox8: https://github.com/Kajabi/sage/pull/31/commits/2f5223b42aeba82579bd0912e5649fd3e1106ff2